pkg/core/app/docker/docker.go

```diff
@@ -518,3 +518,23 @@ func (idc *Impl) SetKeployNetwork(c *Compose) (*NetworkInfo, error) {
 	}
 	return networkInfo, nil
 }
+
+// IsContainerRunning check if the container is already running or not, required for docker start command.
+func (idc *Impl) IsContainerRunning(containerName string) (bool, error) {
+
+	ctx, cancel := context.WithTimeout(context.Background(), idc.timeoutForDockerQuery)
+	defer cancel()
+
+	containerJSON, err := idc.ContainerInspect(ctx, containerName)
+	if err != nil {
+		if nativeDockerClient.IsErrNotFound(err) {
```

Comment:
Add a error log here that the container is not found.

Reply:
Refactored